### DISCUSSION TITLE
Support Yarn 2 ZipFS

### DISCUSSION THREAD

**Discussion by mortyccp** [helix-editor/helix#10197] [Ideas]
We might need to have similar thing like [ZipFS](https://marketplace.visualstudio.com/items?itemName=arcanis.vscode-zipfs) for VSCode to support Yarn 2 ZipFS https://yarnpkg.com/getting-started/editor-sdks#vscode

**Comment by gshokanov** [comment]
I'm interested in this as well, this is a blocker for me to adopt helix as my main editor. I could take a stab at this provided that the core team wants this implemented
